purchase_order/purchase_order.tpl

```diff
@@ -299,15 +299,15 @@ Expected: {{Order.arrivalDate|correcttimezone|date ("m/d/y")}}<br />
 {% endif %}
 <td></td>
 <td></td>
-<td class="money">{{Order.MetaData.subtotal|money}}</td>
+<td class="money">{{Order.MetaData.subtotal|format_currency}}</td>
 </tr>
 {% if Order.MetaData.totalDiscount > 0 %}
 <tr class="minor">
 <th>Discount</th>
 <th></th>
 <td></td>
 <td></td>
-<td class="money">{{Order.MetaData.totalDiscount|getinverse|money}}</td>
+<td class="money">{{Order.MetaData.totalDiscount|format_currency(true)}}</td>
```

Comment:
Could you use a hash as an argument here? it's a lot clearer if someone reading this knows what true stands for example

```javascript
format_currency({invert: true})
```